apps/dc_tools/tests/data/ga_ls8c_ard_3-1-1_088080_2020-05-25_final.stac-item.json

```diff
@@ -0,0 +1,824 @@
+                    -29.485243884787124
+                ],
+                [
+                    153.4214079550725,
+                    -29.006049156688324
+                ],
+                [
+                    153.54919382814404,
+                    -28.52833499498428
+                ],
+                [
+                    153.73227439530447,
+                    -27.837474564334862
+                ],
+                [
+                    153.7376217743478,
+                    -27.817269375728678
+                ],
+                [
+                    153.73791928400988,
+                    -27.816894407110592
+                ],
+                [
+                    153.7385878898377,
+                    -27.81701549471511
+                ]
+            ]
+        ]
+    },
+    "properties": {
+        "datetime": "2020-05-25T23:35:47.745731Z",
+        "dea:dataset_maturity": "final",
+        "end_datetime": "2020-05-25T23:36:02.557719Z",
+        "start_datetime": "2020-05-25T23:35:32.815426Z",
+        "eo:cloud_cover": 42.42102985910952,
+        "gsd": 15.0,
+        "instruments": [
+            "oli",
+            "tirs"
+        ],
+        "platform": "landsat-8",
+        "view:sun_azimuth": 34.22994171,
+        "view:sun_elevation": 31.7895917,
+        "fmask:clear": 1.0427704509544131,
+        "fmask:cloud": 42.42102985910952,
+        "fmask:cloud_shadow": 2.919727915379625,
+        "fmask:snow": 0.00034972190505407335,
+        "fmask:water": 53.61612205265139,
+        "gqa:abs_iterative_mean_x": 0.17,
+        "gqa:abs_iterative_mean_xy": 0.24,
+        "gqa:abs_iterative_mean_y": 0.16,
+        "gqa:abs_x": 0.28,
+        "gqa:abs_xy": 0.4,
+        "gqa:abs_y": 0.28,
+        "gqa:cep90": 0.4,
+        "gqa:iterative_mean_x": 0.03,
+        "gqa:iterative_mean_xy": 0.12,
+        "gqa:iterative_mean_y": 0.12,
+        "gqa:iterative_stddev_x": 0.21,
+        "gqa:iterative_stddev_xy": 0.26,
+        "gqa:iterative_stddev_y": 0.15,
+        "gqa:mean_x": -0.08,
+        "gqa:mean_xy": 0.15,
+        "gqa:mean_y": 0.12,
+        "gqa:stddev_x": 0.42,
+        "gqa:stddev_xy": 0.58,
+        "gqa:stddev_y": 0.4,
+        "landsat:collection_category": "T1",
+        "landsat:collection_number": 1,
+        "landsat:landsat_product_id": "LC08_L1TP_088080_20200525_20200608_01_T1",
+        "landsat:landsat_scene_id": "LC80880802020146LGN00",
+        "landsat:wrs_path": 88,
+        "landsat:wrs_row": 80,
+        "odc:dataset_version": "3.1.0",
+        "odc:file_format": "GeoTIFF",
+        "odc:processing_datetime": "2020-06-18T07:07:06.303867Z",
+        "odc:producer": "ga.gov.au",
+        "odc:product_family": "ard",
+        "odc:region_code": "088080",
+        "odc:product": "ga_ls8c_ard_3",
+        "odc:lineage": {
+            "level1": [
+                "b5f234fe-bba8-5483-9bc0-250360d429cf"
+            ]
+        },
+        "proj:code": 32656,
```

Comment:
Don't think this file passes pystac's `validate()`, this should be `"proj:code": "EPSG:32656",`.

Reply:
Yep, this was a quick and dirty first pass at the end of the day.  Have to double check and clean up a few things.

Reply:
Yes, I totally missed this. thanks.